### PR TITLE
Make recovery words copyable

### PR DESCRIPTION
**Pull request by ichthus1604**
- Enables right-click context menu in the recovery words screen with the "Copy to Clipboard" option.
 - Clears the recovery words from clipboard if they're still there after 30 seconds.

### CODE REVIEW

**Review by YohDeadfall — APPROVED** [review]

File: WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs [hunk 1]
```diff
@@ -37,6 +46,32 @@ private void OnCancel()
...
+		}
+
+		var words =
+			MnemonicWords.Select(x => x.Word).ToArray();
+
+		var text = string.Join(", ", words);
+
+		await Application.Current.Clipboard.SetTextAsync(text);
+
+		Observable.Timer(TimeSpan.FromSeconds(30))
```
Comment (SuperJMN) [inline WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs:63]: Let me see if I understand what goes one here :) 
The rationale here is to empty the contents of the clipboard 30 seconds after being copied, right?

File: WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs [hunk 2]
```diff
@@ -37,6 +46,32 @@ private void OnCancel()
...
+				  .ObserveOn(RxApp.MainThreadScheduler)
+				  .SubscribeAsync(async _ =>
+					{
+						var currentText = await Application.Current.Clipboard.GetTextAsync();
+						if (currentText == text)
+						{
+							await Application.Current.Clipboard.ClearAsync();
+						}
+					});
```
Comment (SuperJMN) [inline WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs:72]: Let me see if I understand what goes one here :)
The rationale here is to empty the contents of the clipboard 30 seconds after being copied, right?

Comment (ichthus1604) [reply WalletWasabi.Fluent/ViewModels/AddWallet/Create/RecoveryWordsViewModel.cs:72]: @SuperJMN exactly.

**Review by RolandUI — APPROVED** [review]
LGTM

**Pull request merged**